### PR TITLE
MGDOBR-312 Creating a SlackAction without parameters gives an Internal server error

### PR DESCRIPTION
**Pull request by lucamolteni**
[Creating a SlackAction without parameters gives an Internal server error](https://issues.redhat.com/browse/MGDOBR-312) 

Please make sure that your PR meets the following requirements:

- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config)
- [x] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [x] All new functionality is tested
- [x] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

* <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

* <b>Rebase the pull request</b>  
  Comment with: `/rebase`.

</details>

### CODE REVIEW

**Review by sutaakar — APPROVED** [review]

**Comment by sutaakar** [comment]
@lucamolteni thinking whether we should revisit also error messages for other actions.

File: manager/src/main/java/com/redhat/service/bridge/manager/actions/connectors/SlackActionValidator.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 public class SlackActionValidator implements ActionParameterValidator {
 
     public static final String EMPTY_PARAMETERS_MESSAGE =
-            String.format("Needed parameters: %s,%s", SlackAction.CHANNEL_PARAMETER, SlackAction.WEBHOOK_URL_PARAMETER);
+            String.format("Missing 'parameter' field with the following parameters: %s,%s", SlackAction.CHANNEL_PARAMETER, SlackAction.WEBHOOK_URL_PARAMETER);
```
Comment (r00ta) [inline manager/src/main/java/com/redhat/service/bridge/manager/actions/connectors/SlackActionValidator.java:15]:
```suggestion
            String.format("Missing 'parameters' field with the following parameters: %s,%s", SlackAction.CHANNEL_PARAMETER, SlackAction.WEBHOOK_URL_PARAMETER);
```

**Review by r00ta — CHANGES_REQUESTED** [review]

**Comment by r00ta** [comment]
Do we have a ticket to fix all the other actions? If you try to create any other action without the `parameters` attribute you are going to face the same problem

**Comment by lucamolteni** [comment]
@r00ta @sutaakar by putting @Valid on the `BaseAction` we don't need to specify the validation on each kind of request, WDYT?

**Review by r00ta — APPROVED** [review]

**Review by sutaakar — APPROVED** [review]

**Pull request merged**